main.js

```diff
@@ -26,13 +30,33 @@ const getPrompt = () => {
     console.log( isRampNumber(answer) );
     getPrompt();
   });
+
 }
 
 // Unit Tests
 // to use them run the command: npm test main.js
 // to close them ctrl + C
 
 
```

Comment:
Had to delete some code over in the main js file in order to merge it. Git hub wouldn't let me merge unless i delete the 
"=======" and the ">>>>>>>chase" lines that you added., but thats all i deleted in order to merge. html and css look clean. Good Job!!!